astropy/units/tests/test_units.py

```diff
@@ -623,7 +623,7 @@ def test_fractional_powers():
     assert x.powers[0].denominator == 7
 
     x = u.cm ** Fraction(1, 2) * u.cm ** Fraction(2, 3)
-    assert type(x.powers[0]) == Fraction
+    assert isinstance(x.powers[0], Fraction)
```

Comment:
The original version is full of this ``isinstance(x..., Fraction)`` checks (e.g. a few lines above). Do you think we should change all of those, too, or leave this as is to be consistent?